.github/workflows/package-client.yml

```diff
@@ -292,6 +292,8 @@ jobs:
       - name: Build Electron bindings
         run: npm run build:release
         working-directory: bindings/electron
+        env:
+          LIBPARSEC_FORCE_ZSTD: real
```

Comment:
Add a comment `# Be explicit about ZSTD implementation to avoid warning on Windows&macOS`

Reply:
done